File: codelearn.io/Training/Python/CountBitsPrime.py
def is_prime(n):
    if n <= 1:
        return False 
    for i in range(2, n):
        if n % i == 0: 
            return False  
    return True


def count_bits_prime(L, R):
    count = 0
    
    for b in range(L, R+1):
       if is_prime(f'{b:08b}'.count("1")):
           count += 1

    return count

File: codelearn.io/Training/C++/CountBitsPrime.cpp
#include <math.h>


bool is_prime(int n)
{
    if (n <= 1) return false;
    for (int i=2; i <= sqrt(n); i++)
        if (n % i == 0) return false;
        
    return true;
}


int decimal_to_binary_and_count(int n)
{
    int count=0;
   
    while (n != 0)
    {
        if (n & 0x01 == 1)
            count++;
        n = n >> 1;
    }

    return count;
}


int countBitsPrime(int L, int R)
{
    int count=0;
    for (int i=L; i <= R; i++)
        if (is_prime(decimal_to_binary_and_count(i)))
            count++;
    return count;
}